Exam/CT/src/exercice2/MaskedRaster.java

```diff
@@ -0,0 +1,33 @@
+package exercice2;
+
+public class MaskedRaster {
+	public static int[][] bits;
+	public static boolean[][] exclusionMask;
+	
+	private MaskedRaster(int[][] input) {
+		bits = input;
+		initMask();
+	}
+	
+	private void initMask() {
+		//exclusionMask = ;
+	}
+	
+	public int[][] getBits() {
+		int n = exclusionMask.length;
+        int m = exclusionMask[0].length;
+        int [][] result = null;
```

Comment:
WTF???!!! How can you use something like `result[i][j] = bits[i][j];` on line24 if you don't initialize properly `result`?